source/_docs/email.md

```diff
@@ -35,17 +61,54 @@ While many providers have standalone modules/plugins, you may want or need to in
 
     <div class="alert alert-info" role="alert">
       <h4 class="info">Note</h4>
-        <p>Support for Drupal 8 is not yet available for the <a href="https://www.drupal.org/project/smtp"> SMTP Authentication Support</a> module. However, <a href="https://groups.google.com/a/pantheon.io/forum/#!topic/power-users/HxvK7T0MPEM"> some users</a> have reported success with the pre-release version. For details, see <a href="/docs/guides/sendgrid/#sendgrid-smtp-integration"> this article</a>.</p>
+        <p>Support for Drupal 8 is in Beta. <a href="https://www.drupal.org/project/smtp"> SMTP Authentication Support</a> module.</p>
```

Comment:
This line ends with a fragment. Not sure if it should be restructured or removed, as the same link lives above.